common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/AuthorizationStrategy.java

```diff
@@ -73,12 +73,12 @@ protected void launchIntent(@NonNull Intent intent) {
         final Fragment fragment = mReferencedFragment.get();
 
         if (fragment != null) {
-            final AuthorizationFragment authFragment = AuthorizationActivity.getAuthorizationFragmentFromStartIntent(intent);
+            final Fragment authFragment = AuthorizationActivityFactory.getAuthorizationFragmentFromStartIntent(intent);
 
             fragment.getFragmentManager()
                     .beginTransaction()
                     .setTransitionStyle(FragmentTransaction.TRANSIT_FRAGMENT_FADE)
-                    .add(fragment.getId(), authFragment, AuthorizationFragment.class.getName())
+                    .add(fragment.getId(), authFragment, Fragment.class.getName())
```

Comment:
instead of doing `Fragment.class.getName()` can we do the following?

`authFragment.getClass().getName()`

Reply:
I"ll check if we're even using this... and again i'll remove from the existing code and this code... if not.